### PR TITLE
Adds VV option to set item's material

### PR DESCRIPTION
**Pull request by comma**
## Description of changes
Adds an option to VV dropdown menu to set material of the item

## Why and what will this PR improve
Beats defining bunch of debug subtypes for testing material things 

## Authorship
Me

## Changelog
N/A

### CODE REVIEW

File: code/game/objects/item_materials.dm [hunk 1]
```diff
@@ -98,8 +98,10 @@
 		throwforce = round(throwforce)
 		attack_cooldown += material.get_attack_cooldown()
 
-/obj/item/proc/set_material(var/new_material)
+/obj/item/proc/set_material(var/decl/material/new_material)
 	if(new_material)
+		if(!ispath(new_material))
```
Comment (MistakeNot4892) [inline code/game/objects/item_materials.dm]: This means that the proc is going to fail if you give it a param of the defined type. That feels like a footgun. Maybe `!ispath(new_material) && !istype(new_material)`?

Comment (comma) [reply code/game/objects/item_materials.dm]: It doesn't handle getting an object instead of a path, since it does GET_DECL on it and gets null

Comment (comma) [reply code/game/objects/item_materials.dm]: Actually axed this change from this PR, bit too big for a simple debug verb PR imo

**Comment by comma** [comment]
Fixes #2304

**Comment by comma** [comment]
Removed sword stuff, gonna do a separate fix PR with that

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**